src/stdlib/native/int.no

```diff
@@ -18,10 +19,90 @@ fn Uint8.new(val: Field) -> Uint8 {
     };
 }
 
+// u16
+struct Uint16 {
+    inner: Field,
+    bit_len: Field,
```

Comment:
as I said elsewhere, I don't think we should carry that in the struct and we should just make sure we use `16` as argument in the right places

Reply:
removed